Towny/src/main/java/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -507,6 +507,18 @@ public static StatusScreen getStatus(Nation nation, CommandSender sender) {
 						.append(Component.newline())
 						.append(translator.component("status_hover_click_for_more"))),
 				ClickEvent.runCommand("/towny:nation enemylist " + nation.getName()));
+		
+		// Sanctioned Towns [3]: Prague, Berlin, Vienna

```

Comment:
This comment should look like so: `// [Sanctioned Towns] + hover` (Yes I know most of the other comments are incorrect.)

Reply:
done